docs/source/common_issues.rst

```diff
@@ -131,6 +131,30 @@ The second line is now fine, since the ignore comment causes the name
     if we did have a stub available for ``frobnicate`` then mypy would
     ignore the ``# type: ignore`` comment and typecheck the stub as usual.
 
+A ``# type: ignore`` comment at the top of a module (before any statements,
```

Comment:
This hardly qualifies as "locally" silencing the checker, so I think it doesn't belong in this section.

I would make it a new section titled "Ignoring a whole file" to be inserted after this section (i.e. just before the section named "Unexpected errors about 'None' and/or 'Optional' types").

Reply:
I've broken it out, as requested.